Content.Server/Botany/Components/SeedComponent.cs

```diff
@@ -15,6 +15,12 @@ public sealed partial class SeedComponent : SharedSeedComponent
         [DataField("seed")]
         public SeedData? Seed;
 
+        /// <summary>
+        ///     The health value of a plant resulting from this seed. If null defaults to the plant's Endurance.
+        /// </summary>
+        [DataField("health")]
```

Comment:
The other values in the file also declare the fields explicitly, so I think it fits.

Reply:
While you're right that the other fields do this, this is because the old code hasn't been updated to meet new conventions. The datafield attribute was recently changed to infer datafield names. The code your seeing is from before it could do this. It would be better if these conventions were updated on a website somewhere, but until then, our job as maintainers is to make sure the code that gets in has current conventions applied.